environment/toolchain/v1.sh

```diff
@@ -22,31 +22,6 @@ SWIG_VERSION=3.0.12 # used only for LLVM compilation
 # check for installed dependencies
 DISTRO="$( egrep '^(VERSION_)?ID=' /etc/os-release | sort | cut -d '=' -f 2- | sed 's/"//g' | paste -s -d '-' )"
 case "$DISTRO" in
-    debian-9)
```

Comment:
This hole block can stay because `toolchain-v1` could stay as is, or we can remove the whole file instead (previous git commits have the file)?

Reply:
Ok I will leave it as is